### DISCUSSION TITLE
Why the emoji icons puts a new line in text?

### DISCUSSION THREAD

**Discussion by javierfs** [HEIGE-PCloud/DoIt#191] [Q&A] [answered]
I am wondering why the icons are centered and putting a new line after being displayed: https://www.visualdatalytics.com/about/

**Comment by HEIGE-PCloud** [comment, marked as answer]
It is a bug relate to `twemoji`. You can either turn it off `twemoji = false` in `config.toml`, or update to the latest version which I have fixed it, or add these lines of CSS

```css
.emoji {
  display: inline-block;
}
```

to fix it yourself.

**Reply by javierfs** [reply]
Thanks a lot! I disabled the twemoji and now is working properly :D
